packages/edit-post/src/hooks/useTextLevelShortcuts.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
The file name should be kebab case instead of camel case 😄 

(if you decide to keep it)